pkg/kubectl/cmd/cmd_test.go

```diff
@@ -204,43 +203,3 @@ func TestClientVersions(t *testing.T) {
 		}
 	}
 }
-
-func ExamplePrintReplicationController() {
```

Comment:
Ah, I didn't notice that either. If you'd like to add a separate commit
that nukes it (since it's unrelated), be my guest. Or one that turns it
into a real test.

On Tue, Feb 24, 2015 at 7:13 AM, Rafał Sokołowski <notifications@github.com>
wrote:

> In pkg/kubectl/cmd/cmd_test.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/4267#discussion_r25258792>
> :
>
> > @@ -204,43 +203,3 @@ func TestClientVersions(t *testing.T) {
> >  		}
> >  	}
> >  }
> > -
> > -func ExamplePrintReplicationController() {
>
> For the sake of making progress on the PR (and also because the change was
> unrelated) I've reverted change in this file.
> The code is dead since the function is not called anywhere from the
> repository, plus this is not a test because it is missing Test prefix and
> doesn't take *testing.Test as an argument (http://golang.org/pkg/testing/).
> Even if such conditions did hold it still wouldn't test anything (except
> that the executed code doesn't crash) since it just prints created
> replication controller to Stdout and doesn't verify the output.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/4267/files#r25258792>
> .
>